transformations/negate_strengthen/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pattern3==3.0.0
+pandas==1.2.5
+spacy==2.2.4
```

Comment:
Please take a master pull, we don't need spacy here now.

Reply:
Ok, done!